minutes/minutes-2015-02-12.md

```diff
@@ -0,0 +1,93 @@
+* 	RS/AP/KL: Fee subcommittee
+* 	GVW: Talk to Andy Terrel about data privacy
+
+
+### Old business:
+  - Old minutes are now online
+  - JW: made contact with Leah Silen
+  - Katy/Matt has sent email to affiliate/partners to have them point out contact people for each institution
+  - Interim members are no longer on board email list nor on private GitHub repo. Tracy Teal is still on due to financial considerations.
+  - Train The Trainers (TTT) events will as before need approval by commite since it is in the bylaws
+
+### Fees for TTT events and workshops
+  - For TTT we can and should go 10k with most folks without loss of interest (need to adjust it in the long run). Currently scheduled TTT event in collaboration with (what in Ottawa?) is set at 5K, however, and will not be changed.
+  - Concerning fees for workshops (and TTT events), it should be noted that money is not that readily available outside US/EU. Wavers have and are given, but it should be stated clearly online in which cases these can be given. A two-tier model was also discussed, however no clear consensus was reached. JW suggested having a form available online to fill out, KL suggested showing examples of when waivers could be given. RS noted that for some institutions it is needed to state the fees from the onset to avoid confusion. A subcommitee will be formed to attend to figuring this out, Aleksandra, Karin and Raniere will discuss further.
+
+### Partners and Affiliates
+  - A Memorandum of Understanding has been received from Mozilla Science Foundation (MSL). The wording of this MoU needs to be agreed on by both sides. The Steering Committee will work together from our side with MSL to get this settled. It should be taken into consideration that MSL is not the only organization that we will be collaborating with in the way we presently collaborate with MSL.
+
+### Data Privacy
+  - GW explained that we are currently storing the name, email address and workshop attended of all workshop participants. We store some more data about instructors. This data is very important for both post-workshop contact and for metrics used in contact with partners and funding bodies.
+  - GW: We presently store this information on an opt-out basis, and are hence breaking laws in several countries. We need to incorporate an opt-in policy and procedure for both storing of data, allowing contact and allowing for sharing information with partners. JW pointed out that there is a risk for losing data with an opt-in policy, and that we might consider ways of encouraging opt-in by for instance sending around list during the workshop itself. We also need to figure out what kind of data we need, and consider the possibility of storing anonymized data.
+  - GW: We are currently developing a workshop managing tool called AMY, which will include handling of this kind of data. GW estimates that there is about 2 days worth of development time needed to resolve privacy issues in AMY.
```

Comment:
What privacy issues need to be resolved in amy?  Opt-in vs. opt-out seems like a collection-time issue, and it seems like we're [currently collecting this information in Eventbrite or by hand][1].  If this is something that can get fixed in amy, it's probably worth opening an amy issue to plan the resolution.

[1]: https://github.com/swcarpentry/amy/issues/27#issuecomment-67974887